Makefile

```diff
@@ -100,10 +100,10 @@ $(HOST_BUILD_DIR)/crc-embedder: $(SOURCES)
 cross: $(BUILD_DIR)/macos-amd64/crc $(BUILD_DIR)/linux-amd64/crc $(BUILD_DIR)/windows-amd64/crc.exe
 
 .PHONY: containerized ## Cross compile from container
-containerized:
+containerized: clean
 	${CONTAINER_RUNTIME} build -t crc-build .
```

Comment:
Still feels that this could get squashed in the initial Dockerfile addition?

Reply:
this is not squashed in that commit as it handles the `COPY` that would else push `out` into the image, but being prevents the build due to a Permission denied, as the go-toolset image uses `1001` as the yuser instead of `root`,

Reply:
https://catalog.redhat.com/software/containers/ubi8/go-toolset/5ce8713aac3db925c03774d1?container-tabs=dockerfile&gti-tabs=unauthenticated

Reply:
But you add a new Containerfile which initially uses the ubi8 image, and subsequently moves to use the go-toolset image. I don't think it's very valuable to record that change in our history, this new file could use the go-toolset image from the start? Especially as the 'move to go-toolset' commit has no commit log, which would indicate it's a non-interesting change?